stock_shipment_management/model/shipment_plan.py

```diff
@@ -207,11 +207,12 @@ def _get_arrival_picking_ids(self):
             shipment.arrival_picking_ids = [(6, 0, pickings[shipment.id])]
 
     @api.one
-    @api.depends('departure_picking_ids.volume')
+    @api.depends('departure_move_ids.product_id',
+                 'departure_move_ids.product_id.volume')
     def _compute_volume(self):
-        volume = 0
-        for picking in self.departure_picking_ids:
-            volume += picking.volume or 0.0
+        volume = 0.0
+        for move in self.departure_move_ids:
```

Comment:
You can do: `sum(x.product_id.volume for x in self.departure_move_ids)`

There's no care about null value, because they are returned already as 0.0.